internal/pkg/agent/application/actions/handlers/handler_action_policy_change_nofips_test.go

```diff
@@ -0,0 +1,238 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+//go:build !requirefips
```

Comment:
What happens here and in the other scenarios when the functionality is called in `fips` mode? Is the `handler` code restricted in fips mode?

Reply:
Handling the encrypted private key returns an error in FIPS mode; this should cause the policy change action to fail